apis/Google.Cloud.Diagnostics.AspNet/docs/index.md

```diff
@@ -26,12 +26,26 @@ installed, run the following command in a Google Cloud SDK Shell:
 
 ## Registering Error Reporting (Web Api)
 
+Reports error events to the Stackdriver Error Reporting API via the Stackdriver Logging API (default
+and suggested method).  For more information see
+["Formatting Log Error Messages"](https://cloud.google.com/error-reporting/docs/formatting-error-messages).
```

Comment:
"Stackdriver Error Reporting API via the Stackdriver Logging API" is a little confusing, especially since the alternative also mentions "Stackdriver Error Reporting API".